lib/grovepi/relay.ex

```diff
@@ -0,0 +1,48 @@
+defmodule GrovePi.Relay do
+  @moduledoc """
+  Conveniences for controlling a [GrovePi Relay](http://wiki.seeed.cc/Grove-Relay/).
+  The relay should be connected to a Digital port (e.g. D3). The relay can
+  be connected to something with a larger load, i.e appliance or desk lamp,
+  which can then be controlled by the
+  [GrovePi](https://www.dexterindustries.com/grovepi/).
+
+  ```elixir
+  iex> alias GrovePi.Relay
+  GrovePi.Relay
+  iex> pin = 3
+  3
+  iex> Relay.start(pin)
+  :ok
+  iex> Relay.on(pin)
+  :ok
+  iex> Relay.off(pin)
+  :ok
+  ```
+  """
+
+  alias GrovePi.Digital
+
+  @doc """
+  Turns off the appliance, lamp, etc. connected to the relay.
+  """
+  @spec off(GrovePi.pin) :: :ok | {:error, term}
+  def off(pin) do
+    Digital.write(pin, 0)
+  end
+
+  @doc """
+  Turns on the appliance, lamp, etc. connected to the relay.
+  """
+  @spec on(GrovePi.pin) :: :ok | {:error, term}
+  def on(pin) do
+    Digital.write(pin, 1)
+  end
+
+  @doc """
+  Sets the pin mode to output. Required prior to using `on/1` or `off/1`.
+  """
+  @spec start(GrovePi.pin) :: :ok | {:error, term}
+  def start(pin) do
```

Comment:
I’m not sure if start is the best name. I like that all the components have a start but it could be confusing since it isn’t a process. What do you guys think?

Reply:
I was just about to add this comment:

Also, I don't really like the name for `GrovePi.Relay.start/1` (this goes for `GrovePi.RGBLCD.start` and GrovePi.PivotPi.start` as well), since it is just setting up the device.

Reply:
What about `GrovePi.Relay.setup`?